dev/mariadb/latest/start.sh

```diff
@@ -3,14 +3,12 @@ set -Eeuxo pipefail # https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo
 cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")" # https://stackoverflow.com/a/17744637
 
 
-docker-compose -p sequelize-mariadb-103 down --remove-orphans
-docker-compose -p sequelize-mariadb-103 up -d
+docker-compose -p sequelize-mariadb-latest down --remove-orphans
+docker-compose -p sequelize-mariadb-latest up -d
 
-./../../wait-until-healthy.sh sequelize-mariadb-103
+./../../wait-until-healthy.sh sequelize-mariadb-latest
 
-docker exec sequelize-mariadb-103 \
+docker exec sequelize-mariadb-latest \
   mysql --host 127.0.0.1 --port 3306 -uroot -psequelize_test -e "GRANT ALL ON *.* TO 'sequelize_test'@'%' with grant option; FLUSH PRIVILEGES;"
 
 DIALECT=mariadb ts-node ../../check-connection.ts
-
-echo "Local MariaDB-10.3 instance is ready for Sequelize tests."
```

Comment:
I removed this message because a similar one has been added to check-connection instead